adagrams/game.py

```diff
@@ -1,13 +1,100 @@
 from random import randint
 
+
 def draw_letters():
-    pass
+    #copy letter_pool from test and change it from dic to list
+    #new variable
+    letter_pool = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'B', 'B', 'C', 'C', 'D', 'D', 'D', 'D',
+ 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'G', 'G', 'G',
+ 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'J', 'K', 'L', 'L', 'L', 'L',
+ 'M', 'M', 'N', 'N', 'N', 'N', 'N', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P',
+ 'P', 'Q', 'R', 'R', 'R', 'R', 'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T',
+ 'T', 'U', 'U', 'U', 'U', 'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z']
+
+    drawn_letters = []
+    #using the row with only from 10 letters that in the letterpool
+    for i in range(10):
+        if len(letter_pool) == 0:
+           break
+
+        index = randint(0, len(letter_pool) - 1)  
+        drawn_letters.append(letter_pool[index])  
+        letter_pool.pop(index)  
+
+    return drawn_letters 
 
 def uses_available_letters(word, letter_bank):
-    pass
+    #create 2 new variables
+   
+    letters_bank_copy = list(letter_bank)
+    word = word.upper()
+    for letter in word:
```

Comment:
```suggestion
    for letter in word.upper():
```
Prefer to call `upper` directly on `word` in the for loop so we don't need to write an extra line of code. More lines of code make a codebase more difficult to maintain and makes for more places a bug can be accidentally introduced.